packages/query/src/bitcoin/address/utxos-by-address.hooks.ts

```diff
@@ -71,13 +71,20 @@ export function useNativeSegwitUtxosByAddress({
   filterPendingTxsUtxos,
   filterRunesUtxos,
 }: UseFilterUtxosByAddressArgs) {
+  const client = useBitcoinClient();
+
+  const initialUtxosQuery = useQuery({
+    ...createGetUtxosByAddressQueryOptions({ address, client }),
+  });
```

Comment:
Think we should make a local wrapper component than using directly in the component?